meta/meta-node/Cargo.toml

```diff
@@ -9,31 +9,34 @@ futures = '0.3.1'
-sc-client-api = { default-features = false, git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-2021-12', version = '4.0.0-dev' }
-sc-consensus = {git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-2021-12', version = '0.10.0-dev'}
-sc-consensus-manual-seal = {git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-2021-12', version = '0.10.0-dev'}
-sc-executor = {git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-2021-12', version = '0.10.0-dev'}
-sc-network = {git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-2021-12', version = '0.10.0-dev'}
-sc-rpc = { git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-2021-12', version = '4.0.0-dev' }
-sc-rpc-api = {git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-2021-12', version = '0.10.0-dev'}
-sc-service = {git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-2021-12', version = '0.10.0-dev'}
-sc-transaction-pool = { git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-2021-12', version = '4.0.0-dev' }
-sc-transaction-pool-api = { git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-2021-12', version = '4.0.0-dev' }
-sc-telemetry = { git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-2021-12', version = '4.0.0-dev' }
-sp-api = { default-features = false, git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-2021-12', version = '4.0.0-dev' }
-sp-block-builder = { default-features = false, git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-2021-12', version = '4.0.0-dev' }
-sp-blockchain = { default-features = false, git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-2021-12', version = '4.0.0-dev' }
-sp-consensus = {git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-2021-12', version = '0.10.0-dev'}
-sp-core = { default-features = false, git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-2021-12', version = '4.0.0-dev' }
-sp-inherents = { default-features = false, git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-2021-12', version = '4.0.0-dev' }
-sp-io = { default-features = false, git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-2021-12', version = '4.0.0-dev' }
-sp-runtime = { default-features = false, git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-2021-12', version = '4.0.0-dev' }
-sp-timestamp = { default-features = false, git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-2021-12', version = '4.0.0-dev' }
-substrate-frame-rpc-system = { git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-2021-12', version = '4.0.0-dev' }
+pallet-transaction-payment-rpc = { git = "https://github.com/paritytech/substrate.git", tag = 'monthly-2022-08', version = "4.0.0-dev" }
```

Comment:
Let's remove the `version` where `tag` is present. should be a redundant detail.